README.md

```diff
@@ -94,44 +118,52 @@ It's simplest to create your configuration file as a "raw" resource file in your
 
 ### Step 4: Create an MSAL PublicClientApplication
 
->NOTE: In this example we are creating an instance of MultipleAccountPublicClientApplication, which is designed to work with apps that allow multiple accounts to be used within the same application. If you would like to use SingleAccount mode, refer to the [single vs. multi account documentation](https://docs.microsoft.com/azure/active-directory/develop/single-multi-account). You can also check out the [quickstart](https://docs.microsoft.com/azure/active-directory/develop/quickstart-v2-android) for examples of how this is used.
+There are two modes for MSAL applications:
+
+1. **Multiple Account Mode** (Default): Allows multiple accounts to be used within the same application
+2. **Single Account Mode**: Restricts the application to use only one account at a time
```

Comment:
nit: maybe single mode first? 

(IIRC most apps don't need more than 1 account, so single mode was introduced to be the "simpler" version of multi account mode)

Reply:
Our default mode in configuration is multiple account mode though